addons/l10n_pe/data/account.account.template.csv

```diff
@@ -1,1540 +1,48 @@
-"273610","Activos no corrientes ...- Intangibles / reservas de recursos extraíbles, costo","273610","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"273620","Activos no corrientes ...- Intangibles / reservas de recursos extraíbles, revaluación","273620","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"273910","Activos no corrientes ...- Intangibles / otros activos intangibles, costo","273910","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"273920","Activos no corrientes ...- Intangibles / otros activos intangibles, revaluación","273920","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"274110","Activos no corrientes ...- Activos biológicos / activos biológicos en producción, valor razonable","274110","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"274120","Activos no corrientes ...- Activos biológicos / activos biológicos en producción, costo","274120","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"274130","Activos no corrientes ...- Activos biológicos / activos biológicos en producción, costos de financiación","274130","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"274210","Activos no corrientes ...- Activos biológicos / activos biológicos en desarrollo, valor razonable","274210","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"274220","Activos no corrientes ...- Activos biológicos / activos biológicos en desarrollo, costo","274220","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"274230","Activos no corrientes ...- Activos biológicos / activos biológicos en desarrollo, costos de financiación","274230","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"275210","Activos no corrientes ...- Depreciación acumulada.../ edificaciones, valor razonable","275210","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"275220","Activos no corrientes ...- Depreciación acumulada.../ edificaciones, costo","275220","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"275230","Activos no corrientes ...- Depreciación acumulada.../ edificaciones, revaluación","275230","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"276210","Activos no corrientes ...- Depreciación acumulada.../ edificaciones, costo de adquisición o construcción ","276210","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"276220","Activos no corrientes ...- Depreciación acumulada.../ edificaciones, revaluación  ","276220","account.data_account_type_current_assets","l10n_pe.pe_chart_template","False"
-"276230","Activos no corrientes ...- Depreciación acumulada.../ edificaciones, costo
```

Comment:
I just added the minimal ones as we did in the past, to comply with a minimal company, I did not want to add them all (almost 200) but I can if you wish.

Reply:
I was trying to add just the minimal accounts to achieve what is demonstrated [here](https://www.odoo.com/documentation/functional/accounting.html):

Reply:
On the other hand, it is also that an SMB should not need to start creating accounts.  (An accountant should have the impression he can start right away)  Maybe the 200 accounts is more appropriate for that?

Reply:
Ok, this will take me a few minuts but that make sense, I will. (**TODO: FIxme**)

**note:** Take care that some of those account are for operations that we do not actually support, then even if the account will exists, we will not have administrative features that will use them left them as simply waiting to be set manually.

Reply:
Hello, 

I added all possible accounts, then nobody need to add new ones.

I will propose a little improvement on the display_name in order to allow the combination with the group, because the account by themselves in some cases have non descriptive and repetitive name in different levels.

Reply:
the change propesed on groups will be managed [here](https://github.com/odoo/odoo/pull/33171)

Reply:
@jco-odoo indeed, that's why I did this other PR:

https://github.com/odoo/odoo/pull/33171

To solve exactly such problem.

Reply:
@nhomar Maybe I should have said the 48 were just fine. (the pr seems problematic for stable)  It is not the first time I see this case where you have the same subaccounts, but I wonder if it really makes sense and if everyone does it that way.  They can not just be grouped to one account?